lib/internal/http2/util.js

```diff
@@ -443,6 +447,9 @@ const assertValidPseudoHeaderTrailer = hideStackFrames((key) => {
   throw new ERR_HTTP2_INVALID_PSEUDOHEADER(key);
 });
 
+const emptyArray = [];
+const kNeverIndexFlag = String.fromCharCode(NGHTTP2_NV_FLAG_NO_INDEX);
+const kNoHeaderFlags = String.fromCharCode(NGHTTP3_NV_FLAG_NONE);
```

Comment:
`NGHTTP2_NV_FLAG_NONE`

Reply:
Yeah, these are typos which just happen to work out correctly ;) Fixed!